lib/registries/addon/overview/-components/overview-topbar/component.ts

```diff
@@ -79,9 +79,7 @@ export default class OverviewTopbar extends Component {
             const errorMessage = this.intl.t('registries.overview.fork.error');
             captureException(e, { errorMessage });
             this.toast.error(getApiErrorMessage(e), errorMessage);
-            throw e;
-        } finally {
-            closeDropdown();
+            // throw e;
```

Comment:
Do we want to throw this error or just remove this line entirely?

Reply:
I will remove it.